scarb/build.rs

```diff
@@ -104,18 +105,25 @@ fn download_core(rev: &str) {
 
     let core_path = PathBuf::from_iter([&out_dir, &format!("core-{}", ident(rev))]);
     if !core_path.is_dir() {
-        let url = format!("https://github.com/starkware-libs/cairo/archive/{rev}.zip");
         let cairo_zip = PathBuf::from_iter([&out_dir, "cairo.zip"]);
 
-        let mut curl = Command::new("curl");
-        curl.args(["--proto", "=https", "--tlsv1.2", "-fL"]);
-        curl.arg("-o");
-        curl.arg(&cairo_zip);
-        curl.arg(&url);
-        eprintln!("{curl:?}");
-        let curl_exit = curl.status().expect("Failed to start curl");
-        if !curl_exit.success() {
-            panic!("Failed to download {url} with curl")
+        if let Ok(cairo_archive) = std::env::var("CAIRO_ARCHIVE") {
+            // Copy archive to `cairo_zip`, without keeping file attributes.
+            let mut src = File::open(&cairo_archive).unwrap();
```

Comment:
Could we add some more explicit message here, so that debugging panicked build would be easier to do? Maybe something like:  
```suggestion
            eprintln!("Copying Cairo archive from `CAIRO_ARCHIVE={cairo_archive}`.");
            let mut src = File::open(&cairo_archive).expect("Failed to open the Cairo archive file.");
```